lib/win_draw_checkers.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module WinDrawCheckers
```

Comment:
Maybe a nitpick, but the naming of this seems a little strange. Someone may easily misunderstand this as related to the game of checkers.